src/biblenotify/core/notifications.py

```diff
@@ -96,7 +98,43 @@ def loadChapter(self, location: str) -> dict:
 
         contents_json = json.loads(contents_string)
 
+        text = self.highlightVerses(contents_json)
+
         return {
-            "text": contents_json["read"][0]["text"],
+            "text": text,
             "place": contents_json["read"][0]["chapter"]
         }
+
+    @Slot(str, result="QVariant")
```

Comment:
This line is not necessary since this method is not used directly in the UI.

Reply:
Oh